src/clarity-angular/datagrid/render/main-renderer.ts

```diff
@@ -84,6 +83,27 @@ export class DatagridMainRenderer implements AfterContentInit, AfterViewInit, On
     private computeDatagridHeight() {
         let value: number = this.domAdapter.computedHeight(this.el.nativeElement);
         this.renderer.setElementStyle(this.el.nativeElement, "height", value + "px");
+        this._heightSet = true;
+    }
+
+    private resetDatagridHeight() {
+        this.renderer.setElementStyle(this.el.nativeElement, "height", "auto");
```

Comment:
I don't think you want `auto`, because that is not actually neutral. It would override a height given through CSS, for instance. The correct reset is: `null` or the empty string `""`. Both should work fine and actually _remove_ the height property from the style inlined on the element.